r/README.md

```diff
@@ -62,6 +63,14 @@ Arrow C++ library first.
 
 ``` r
 library(arrow)
+#> 
```

Comment:
Could you remove this added comment output? It's not helpful IMO. (If you regenerate this with `make doc`, it doesn't appear because the arrow package is already loaded before this code chunk is evaluated.)

Reply:
I see, will do